### PR TITLE
chore: rename develop branch to main

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: tools/release.js [hunk 1]
```diff
@@ -333,23 +332,23 @@ const indicators = new Map([
...
       case "needs a pull":
-        await Prompts.askToPullBranch(MAIN_BRANCH);
+        await Prompts.askToPullBranch("main");
         break;
       case "up-to-date":
         break;
       case "needs to push":
         throw new Error(
-          `Found unpushed commits on "${MAIN_BRANCH}" branch! Can't proceed.`
+          `Found unpushed commits on "${"main"}" branch! Can't proceed.`
```
Comment (sidvishnoi) [inline tools/release.js:351]:
```suggestion
          `Found unpushed commits on "main" branch! Can't proceed.`
```

File: tools/release.js [hunk 2]
```diff
@@ -109,7 +108,7 @@ const Prompts = {
       await this.askQuestion(promptOps);
     } catch (err) {
       const warning = colors.yellow(
-        "🚨 Make sure to run `git up; git checkout develop`"
+        "🚨 Make sure to run `git up; git checkout main`"
```
Comment (sidvishnoi) [inline tools/release.js:111]: What's this `git up` here?

File: .github/workflows/push.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ name: Node CI (push)
 on:
   push:
     branches:
-      - develop
+      - main
       - gh-pages
```
Comment (sidvishnoi) [inline .github/workflows/push.yml:6]: Let's remove this gh-pages while we're here

**Review by sidvishnoi — APPROVED** [review]

File: tools/release.js (hunk 2 above)
Comment (marcoscaceres) [reply tools/release.js:111]: oh yeah, lol... `git up`... that super ol' school, lol. I think it did a `git pull` + rebase or something 

I think it was this... 
https://github.com/git-up/GitUp

Comment (marcoscaceres) [inline tools/release.js:111]:
```suggestion
        "🚨 Make sure to run `git checkout main` and rest any changes."
```

File: tools/release.js [hunk 4]
```diff
@@ -109,7 +108,7 @@ const Prompts = {
       await this.askQuestion(promptOps);
     } catch (err) {
       const warning = colors.yellow(
-        "🚨 Make sure to run `git up; git checkout develop`"
+        "🚨 Make sure to run `git checkout main` and rest any changes."
```
Comment (sidvishnoi) [inline tools/release.js:111]:
```suggestion
        "🚨 Make sure to run `git checkout main` and reset any changes."
```

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**